### DISCUSSION TITLE
Lazyvim & Angular

### DISCUSSION THREAD

**Discussion by MatthieuTricoire** [LazyVim/LazyVim#4419] [Q&A]
Hello everyone,
I'd like to take this opportunity to thank Folke for allowing me to discover Vim without getting too deep into the nitty-gritty and wasting several days tinkering with my config files. It's just amazing :)

I hope you don't mind me asking a quick question. It's probably a misunderstanding on my part, but I recently started using a new framework to prepare for my upcoming apprenticeship, and I'm running into issues with Angular.

In my HTML file, it feels like the LSP is acting up, and autocompletion isn't working correctly.

Here are two examples:
Why is this property identified as text?
![CleanShot 2024-09-09 at 16 20 31](https://github.com/user-attachments/assets/e9d424c4-60b9-4642-ae1a-32404b9ce4a4)

Why the task property is not listed ?
![CleanShot 2024-09-09 at 16 21 41](https://github.com/user-attachments/assets/ff628960-e6cd-44c0-a422-f8c800285f3d)

I’m just starting with Angular, so it might not be a problem but rather a misunderstanding on my part.

**Comment by zodman** [comment]
Ensure you have installed Mason: `angular-language-server` and create a type `Task` instead of `{id:prop, ...}`

I think that functionality comes from LSP, not from lazyvim. 

https://github.com/angular/vscode-ng-language-service

**Comment by MatthieuTricoire** [comment]
Yes you probably right.
But angular-language-server is installed (part of some extra from lazyvim) and is properly attached to the buffer.
The task is already define in a type. I’ll investigate more tomorrow.
Thanks for your reply 🙏

Edit : I've made a fresh install of lazyvim and i notice this warning message during angular configuration : 
![CleanShot 2024-09-10 at 10 19 39](https://github.com/user-attachments/assets/a8e1b940-54ed-41c6-ac3f-2a4951469549)
After forcing mason update, my html file is correctely parsed with treesitter :)
